src/views/WorkoutView.tsx

```diff
@@ -0,0 +1,268 @@
+    return (
+        <div className="flex flex-col items-center w-full max-w-2xl px-4 mx-auto">
+            <h1>Workout</h1>
+
+            {renderCurrentTimer()}
+
+            <div className="w-full max-w-md">
+                {timers.map((timer, index) => (
+                    <div
+                        key={index}
+                        className={`
+            flex
+            justify-between
+            items-center
+            mb-2 
+            p-2 
+            rounded-lg
+            bg-slate-900/50
+            ${timer.state === 'completed' ? 'line-through text-gray-600' : ''}
+            ${hasStarted && index === currentTimerIndex ? 'text-lime-200' : ''}
+          `}
+                    >
+                        <div className="flex flex-col">
+                            <Button
+                                onClick={() => handleMoveUp(index)}
+                                disabled={
+                                    (hasStarted && index <= currentTimerIndex) ||
+                                    timer.state === 'completed' ||
+                                    index === 0 ||
+                                    (index > 0 && (timers[index - 1].state === 'completed' || timers[index - 1].state === 'running'))
+                                }
+                            >
+                                ↑
+                            </Button>
+                            <Button onClick={() => handleMoveDown(index)} disabled={(hasStarted && index <= currentTimerIndex) || timer.state === 'completed' || index === timers.length - 1}>
+                                ↓
+                            </Button>
+                        </div>
+                        <div className="flex flex-col">
+                            <div className="flex-grow text-center">{displayTimerDetails(timer)}</div>
+                            {timer.description && <div className="text-sm text-gray-500 text-center mt-1">{timer.description}</div>}
+                        </div>
+                        <div className="flex flex-col gap-2 ml-4">
+                            <Button onClick={() => editTimer(index)} disabled={(hasStarted && index <= currentTimerIndex) || timer.state === 'completed'}>
+                                Edit
+                            </Button>
+                            <Button onClick={() => removeTimer(index)} disabled={(hasStarted && index <= currentTimerIndex) || timer.state === 'completed'}>
+                                Remove
+                            </Button>
+                        </div>
+                    </div>
+                ))}
+            </div>
+            <div>
+                {timers.length >= MAX_TIMERS ? (
+                    <span className="text-gray-500">
+                        <i>You've added the maximum number of timers.</i>
+                    </span>
+                ) : (
+                    <Button onClick={addTimer}>+ Add Timer</Button>
```

Comment:
Adding a new timer after a workout runs is not working correctly. It adds the timer, but then a pause/play set of buttons show up, with a 0:00 amount of time. If i press play, it starts the workout from the previous timer (the one that was the final timer, before I added the new one), then runs the newly added timer.  

You should suppress the pause/play from showing up, and leave the "reset timer" button showing after add.